### PR TITLE
Added contributing docs

### PR DESCRIPTION
**Pull request by silvererudite**
hi @maxxcrawford and @groovecoder . I noticed the project was missing a contributing docs so I added one. I followed the format on multi-account-containers and added few instructions myself as well.Not sure about the testing part though. Please review and suggest changes where required.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Running
+After you make changes to your local copy. You can test to see your changes in two ways
+1. Install `web-ext` tool and run to see your changes
+2. go to `about:debugging` in your browser and select `load temporary Add-on`
+3. Select `container-facebook` from your source dir and then select the `manifest.json` file to load the extension
+4. you can `reload` after you make new changes to see them in effect.
+
+## Testing
+
+TBD
```
Comment (maxxcrawford) [inline Contributing.md]: Same comment as above. Remove this section and add something to the main README file.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+## Sending Pull Requests
+
+Patches should be submitted as pull requests. When submitting patches as PRs:
+
+- You agree to license your code under the project's open source license (MPL 2.0).
+- Base your branch off the current master (see below for an example workflow).
+- Add both your code and new tests if relevant.
+- Run npm test to make sure all tests still pass.
```
Comment (maxxcrawford) [inline Contributing.md]: Use code notation for any npm commands. 

```suggestion
- Run `npm test` to make sure all tests still pass.
```

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Contributing
+
+Everyone is welcome to contribute to facebook-containers. Reach out to team members if you have questions:
+
+- IRC: #containers on irc.mozilla.org
+- Email: containers@mozilla.com
+
+## Filing bugs
+
+If you find a bug with facebook-container, please file a issue.
```
Comment (maxxcrawford) [inline Contributing.md]:
```suggestion
If you find a bug with Facebook Container, please file a issue.
```

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+1. Visit about:support
+2. Click "Copy raw data to clipboard" and paste into the bug. Alternatively copy the following sections into the issue:
+  - Application Basics
+  - Nightly Features (if you are in nightly)
+  - Extensions
+  - Experimental Features
+3. Include clear steps to reproduce the issue you have experienced.
+4. Include screenshots if possible.
+
+## Installing
```
Comment (maxxcrawford) [inline Contributing.md]: Remove this section. I would like to add documentation for setting the project up to the README itself. (Similar to how MAC documents it.)

File: Contributing.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# Contributing
```
Comment (maxxcrawford) [inline Contributing.md]: Can you make the file name all caps? (CONTRIBUTING)

**Comment by silvererudite** [comment]
@maxxcrawford made the contributing docs identical to MAC's doc.

File: Contributing.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+Everyone is welcome to contribute to facebook-containers. Reach out to team members if you have questions:
```
Comment (maxxcrawford) [inline Contributing.md]:
```suggestion
Everyone is welcome to contribute to Facebook Container. Reach out to team members if you have questions:
```

**Comment by silvererudite** [comment]
@maxxcrawford done

**Comment by dnahol** [comment]
Hi! I didn't see this before adding my own PR for this today: #541 
I made some suggestions in comments.

This relates to issue #487

**Comment by dnahol** [comment]
@silvererudite taking the discussion from #541 here

> However the project still misses a development setup guide in the README like instructions for testing, running etc as the README of Multi Account Containers so better update the README ? This will be a better and more useful contribution. Hope you don't mind.

I hear your suggestion. I'm unsure whether said instructions should go in CONTRIBUTING or in the README. To me it seems like they should go in CONTRIBUTING, here. 

I submitted two comments with suggested edits in my review above. 
What do you think?

**Comment by silvererudite** [comment]
Hey @dnahol ! As per the reviewers previous sugeestions I belie the changes you suggested need to be put in the README just like Multi Account Conatiners repo. That is also why the link to the README was put in the contributing docs. However I wasn't able to update the README that time. If you have the time please add the suggested changes to the README as per the previous reviews here and your ideas. 
I have added the ss of previous review for your reference.
Love your ideas and look forward to interact with you more. 
image
image

**Comment by dnahol** [comment]
Hi @silvererudite ; ah I see that now. Okay will add those changes to the README!

**Pull request merged**